src/core/Raycaster.js

```diff
@@ -127,7 +119,18 @@ Object.assign( Raycaster.prototype, {
 
 	}
 
-} );
+}
+
+Object.assign( Raycaster.prototype, {
 
+	set: function ( origin, direction ) {
```

Comment:
Why is `set()` not defined like all other methods?

Reply:
TBH, I just use the code as before, I am not sure it can work...，I found some code as you say, I change it right now.